osf_pigeon/settings/test.py

```diff
@@ -18,3 +18,6 @@
 
 REG_ID_TEMPLATE = f"osf-registrations-{{guid}}-{ID_VERSION}"
 PROVIDER_ID_TEMPLATE = f"osf-registration-providers-{{provider_id}}-{ID_VERSION}"
+
+PAGING_SEMAPHORE = 5
```

Comment:
These values changed while the tests were offline, so they are adjusted now that they are back.